ByeBoo-iOS/ByeBoo-iOS/Presentation/Feature/Home/ViewModel/CombineTestViewModel.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CombineTestViewModel.swift
+//  ByeBoo-iOS
+//
+//  Created by APPLE on 7/1/25.
+//
+
+import Combine
+import UIKit
+
+final class CombineTestViewModel: ViewModelType {
+    
+    struct Input {
+        let publisher: AnyPublisher<Void, Never>
+    }
```

Comment:
Input의 경우의 수가 많아질 경우 유지보수를 위해 Input을 enum으로 정의해보는 건 어떤가요?
```suggestion
    enum InputAction {
         case first
         case second
    }

    struct Input {
        let publisher: AnyPublisher<InputAction, Never>
    }
```

이렇게 선언하고 transform에서

```
func transform(input: Input) -> Output {
        input.publisher
            .sink { [weak self] action in
                switch action {
                     case .first:
                     case .second:
                 }
            }
            .store(in: &cancellables)
        return Output(result: output.eraseToAnyPublisher())
}
```

이런식으로요!

Reply:
오오오 좋아요..! 
```
struct Input {
    let firstButtonTap: AnyPublisher<Void, Never>
    let secondButtonTap: AnyPublisher<Void, Never>
    let textChanged: AnyPublisher<String, Never>
}
```
원래는 위 코드처럼 구조체 안에 속성을 추가하려고 했는데, 생각해보니 enum으로 구현해야 각 케이스를 하나의 스트림으로 통합할 수 있겠네요! 수정하는 게 좋을 것 같습니다